aten/src/ATen/native/transformers/cuda/attention_backward.cu

```diff
@@ -159,18 +159,32 @@ std::tuple<Tensor, Tensor, Tensor> _scaled_dot_product_cudnn_attention_backward_
     const Tensor& value,
     const Tensor& out,
     const Tensor& logsumexp,
-    const Tensor& cumulative_sequence_length_q,
-    const Tensor& cumulative_sequence_length_k,
-    const int64_t max_seqlen_batch_q,
-    const int64_t max_seqlen_batch_k,
-    double dropout_p,
-    bool is_causal,
     const Tensor& philox_seed,
     const Tensor& philox_offset,
+//    const Tensor& cumulative_sequence_length_q,
```

Comment:
do you know if cudnn supports a ragged version of SDPA?

Reply:
Word from the team is that it does, could be integrated in a follow-up PR